flags/src/main/java/com/yahoo/vespa/flags/Flags.java

```diff
@@ -175,6 +175,12 @@
             "Takes effect at redeployment",
             APPLICATION_ID);
 
+    public static final UnboundBooleanFlag HOST_HARDENING = defineFeatureFlag(
+            "host-hardening", false,
+            "Whether to enable host hardening Linux baseline.",
+            "Typically used in host-admin and takes effect on next tick or on host-admin restart (may vary where used).",
```

Comment:
Where else can it be used?

Reply:
Fixed: I have narrowed the scope to host-admin.